go.mod

```diff
@@ -5,6 +5,7 @@ go 1.13
 require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/blang/semver v3.5.0+incompatible
+	github.com/coredns/corefile-migration/migration v1.0.1
```

Comment:
`github.com/coredns/corefile-migration v1.0.6`?

Reply:
I'm wondering if we should think about some type of automated checking to see if it is out of date, since the current implementation will require us to cut a new release if a newer version of CoreDNS is released prior to us being able to support it (based on the version comparison check)

Reply:
Hmm...yeah I don't know why this didn't pull in the latest release. I was expecting it to pull in v1.0.6
I'll manually fix this. 
@detiber I agree. That was one of the questions I had actually since it seems the Corefile migration is hardcoded in the library. 
Should I create a separate issue to get that going?

Reply:
Please do, I think it'll be important for the short term until we have a way to decouple this a bit better.

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/2599
For reference.